test/utils/XhrNetworkInterceptor.spec.ts

```diff
@@ -372,13 +375,14 @@ describe('Network Interceptor W3C Headers', () => {
     };
     Interceptor.enableInterception();
     Interceptor.setOnDoneCallback((network) => {
-      injectHeaders(network, featureFlags);
-      expect(network.w3cc).toBe(null);
-      expect(network.partialId).toBe(null);
-      expect(network.etst).toBe(null);
-      expect(network.wgeti).toBe(null);
-      expect(network.wceti).toBe(null);
-      expect(network.requestHeaders).not.toHaveProperty('traceparent');
+      injectHeaders(network, featureFlags).then((res) => {
+        expect(res).toBeUndefined();
+        expect(network.w3cc).toBe(null);
+        expect(network.partialId).toBe(null);
+        expect(network.etst).toBe(null);
+        expect(network.wgeti).toBe(null);
+        expect(network.wceti).toBe(null);
+      });
```

Comment:
I think using async/await would be more readable here and in the other similar tests, what do you think?